pallets/fungible-assets/src/lib.rs

```diff
@@ -241,14 +239,15 @@ decl_module! {
         pub fn claim_vesting(origin, identifier: T::Hash, asset_id: T::CurrencyId) -> DispatchResult {
             let who: T::AccountId = ensure_signed(origin)?;
             let current_block_no = <system::Pallet<T>>::block_number();
-
+            assert_eq!(1,0,"code in use");
```

Comment:
I don't know the purpose of this

Reply:
This should not be there!!! Good spoted!